modules/apps/27-interchain-accounts/host/ibc_module.go

```diff
@@ -144,6 +145,7 @@ func (IBCModule) OnAcknowledgementPacket(
 	packet channeltypes.Packet,
 	acknowledgement []byte,
 	relayer sdk.AccAddress,
+	channelVersion string,
```

Comment:
```suggestion
	_ string,
```
I guess we could discard here as well

Reply:
Sure thing. All the other args are also unused (same in quite a few other functions here), should I discard those as well to keep the file consistent?

Reply:
I've changed it for the whole file now :)